client/src/css/Alternative.css

```diff
@@ -1,43 +1,45 @@
 .Alternative {
-  margin: 1rem 0;
-  background: #6b71a5;
-  color: #ffffff;
-  font-size: 1.8rem;
+  margin: 0.5rem 0;
+  background-color: #6B71A5;
+  color: #FFF;
   display: flex;
-  box-shadow: 0px -2px 0px 0px rgba(0, 0, 0, 0.25) inset,
-              0px 2px 1px 0px rgba(0, 0, 0, 0.2);
+  box-shadow: 0 -0.125rem 0 0 rgba(0, 0, 0, 0.25) inset,
+              0 0.125rem 0.0625rem 0 rgba(0, 0, 0, 0.2);
 }
 
 .Alternative:last-of-type {
   margin-bottom: 0;
 }
 
 .Alternative:hover {
-  background: #8d93c7;
+  background-color: #8D93C7;
```

Comment:
Uppercase hex:
I don't like this. The way I pick colors is using Chrome dev tools and this defaults to lowercase. 
Meaning that every time I want to change a hex color I have to manually rewrite it to uppercase or end up with a mix. 

background -> background-color:
I don't see any reason for this change, but I don't really care that much either.